priv/templates/boot.eex

```diff
@@ -44,7 +72,11 @@ REL_LIB_DIR="$RELEASE_ROOT_DIR/lib"
 ERL_OPTS="<%= release.profile.erl_opts %>"
 # When stdout is piped to a file, this is the directory those files will
 # be stored in. defaults to /log in the release root directory
-RUNNER_LOG_DIR="${RUNNER_LOG_DIR:-$RELEASE_ROOT_DIR/log}"
+
+# RELEASE_MUTABLE_DIR
+RELEASE_MUTABLE_DIR=${RELEASE_MUTABLE_DIR:-$RELEASE_ROOT_DIR/running-environment}
```

Comment:
Also I'm kind of in favor of using `tmp` instead of `running-environment`, but I'm open to being convinced if there is some sort of additional significance to the name.